llama-index-integrations/llms/llama-index-llms-nvidia/tests/test_base_url.py

```
@@ -32,27 +30,20 @@ def mock_local_models(httpx_mock: HTTPXMock, base_url: str):
 @pytest.mark.parametrize(
     "base_url",
     [
-        "localhost",
-        "localhost:8888",
-        "http://localhost:8888/embeddings",
-        "http://0.0.0.0:8888/rankings",
-        "http://localhost:8888/chat/completions",
-        "http://test_url/.../v1",
-        "https://test_url/.../v1",
+        "http://0.0.0.0:8888/chat/completion",
+        "https://0.0.0.0:8888/ranking",
     ],
 )
-def test_base_url_invalid_not_hosted(base_url: str) -> None:
-    with pytest.raises(ValueError) as msg:
-        Interface(base_url=base_url)
-    assert "Invalid base_url" in str(msg.value)
+def test_base_url_invalid_not_hosted(base_url: str, mock_local_models: None) -> None:
```

this changes semantics where being invalid should raise an error